CONTRIBUTING.md

```diff
@@ -23,12 +23,10 @@ Thanks for your interest in contributing to Ionic's documentation! :tada: Check
 In order to run the documentation locally, install the dependencies and run the development server:
 
 ```sh
-$ npm install --legacy-peer-deps
-$ npm start
+npm install
```

Comment:
The translation icon in the top menu bar appears smaller/thinner than main

| main | branch |
| - | - |
| ![image](https://github.com/ionic-team/ionic-docs/assets/2721089/7a51e8b3-f4ec-4055-ae41-84cd06a9095b) | ![image](https://github.com/ionic-team/ionic-docs/assets/2721089/55bdc72f-6f46-47ce-8181-bd5530089354) |

Reply:
I had to hide the default icon that Docusaurus renders and [import a new icon via CSS](https://github.com/ionic-team/ionic-docs/pull/3014/files#diff-035ec64dbe3b95d97cbc279267dee0159d93b22d15e30970d9c53c16ba799a56R296-R298). The new icon has a bolder setup. This feels a bit weird. Do we want to continue with this approach or use the default?

Reply:
The thinner icon is the default that Docusaurus provides. Main also opts out of using the default and uses the import via CSS. The reason seems to be due to wanting a different icon at that time. Here's a screenshot of the imported icon and the default icon on main side by side.

![Screenshot 2023-09-06 at 8 13 51 AM](https://github.com/ionic-team/ionic-docs/assets/13530427/3bc9a27c-5ee1-489c-9727-a347e7b21089) (main on left, Docusaurus on right)

Fast forward to today, Docusaurus has changed it's icon. It matches the one shown in main but the SVG is thinner.

![Screenshot 2023-09-06 at 8 13 56 AM](https://github.com/ionic-team/ionic-docs/assets/13530427/4b54012e-e76d-4c6a-be14-b7520b01bd73)  (main on left, Docusaurus on right)

I tried to match the default to main but couldn't get it so I opted to import main's icon via CSS. Realizing it now, I might be able to import the language icon from Ionicons. It might match the thickness. I'll give this a shot.